atomic-design.md

```diff
@@ -0,0 +1,101 @@
+# Atomic Design Guide
+
+> Helpful guidance and tips for building front-end components atomically.
+
+## What is Atomic Design?
+
+Created by [Brad Frost](http://bradfrost.com/blog/post/atomic-web-design/), Atomic Design is a system for building and maintaining reusable components (we refer to them as elements) for a web platform. By creating simple components, and combining them to build larger and more complex components, we can ensure a consistent design throughout our projects.
+
+## What's an atomic element?
+
+An atomic element is any self contained code pattern that follows the Atomic Design principles. The types of elements are atoms, molecules, organisms, and templates.
+
+### Atoms
+
+- The base unit
+- Think buttons, input fields, and other HTML tags
+- Abstract and not too useful on their own
+
+### Molecules
+
+- It can do something
+- A series of atoms combined to serve a purpose
+
+### Organisms
+
+- Self contained chunks with different functionalities to form a distinct section
+- Think headers, comments section, etc
+
+### Templates
+
+Types of pages that have a particular layout. The primary ones are Landing, Sublanding, Browse, Learn, and Document Detail. They are differentiated primarily by their major layout features, such as whether or not they have a hero, whether the sidebar is on the right or left, etc. Each template allows certain atoms, molecules, and organisms to be included in its major sections.
+
+### How do I know what to make into an element?
```

Comment:
Maybe add a section below this of what to do if it shouldn't be atomized, e.g. whether it should probably be a modifier.